<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employees</class>
 <widget class="QMainWindow" name="Employees">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employees</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/images/Users.png</normaloff>:/images/Users.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="placeholderText">
           <string>Search employees</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ShowInactive">
          <property name="text">
           <string>Show inactive employees</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="HrTab">
           <attribute name="title">
            <string>HR Details</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Employee No:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QLineEdit" name="EmployeeNo"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>First Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QLineEdit" name="FirstName"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>ID No:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QLineEdit" name="IDNo"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Date Of Birth:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1" colspan="2">
             <widget class="QDateEdit" name="DateOfBirth">
              <property name="date">
               <date>
                <year>2012</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="displayFormat">
               <string>dd-MMM-yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Department:</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1" colspan="2">
             <widget class="QComboBox" name="Department"/>
            </item>
            <item row="13" column="0" colspan="3">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Contact Information</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Address:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Tel:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="Tel"/>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="Address"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="13" column="10">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>2</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="15" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>22</width>
                <height>172</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="14" column="0" colspan="3">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Next Of Kin</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0" rowspan="2">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="Name"/>
               </item>
               <item row="1" column="1" rowspan="2">
                <widget class="QLineEdit" name="Relationship"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Relationship:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Tel:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="Tel_2"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Job Title:</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1" colspan="2">
             <widget class="QComboBox" name="JobTitle"/>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Supervisor:</string>
              </property>
             </widget>
            </item>
            <item row="12" column="1" colspan="2">
             <widget class="QComboBox" name="Supervisor"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Other Names:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QLineEdit" name="OtherNames"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>PIN No:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QLineEdit" name="PINNo"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>NHIF No:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QLineEdit" name="NHIFNo"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>NSSF No:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1" colspan="2">
             <widget class="QLineEdit" name="NSSFNo"/>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Date Employed:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1" colspan="2">
             <widget class="QDateEdit" name="DateEmployed">
              <property name="date">
               <date>
                <year>2012</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="displayFormat">
               <string>dd-MMM-yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>Employment Type:</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1" colspan="2">
             <widget class="QComboBox" name="comboBox">
              <property name="editable">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>Casual</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Contract</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Permanent</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="FinanceTab">
           <attribute name="title">
            <string>Finance Details</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Basic Pay:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="BasicPay"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>House Allowance:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="HouseAllowance"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Payment Method:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QComboBox" name="PaymentMethod">
              <property name="editable">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>Cash</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cheque</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Direct Transfer</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Bank Name:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="BankName"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Branch:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="BankBranch"/>
            </item>
            <item row="2" column="4">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>159</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Account Name:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="AccountName"/>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Account No:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLineEdit" name="AccountNo"/>
            </item>
            <item row="4" column="3">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>304</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Advances</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTreeView" name="AdvancesTree">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QToolButton" name="AddAdvance">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="rc.qrc">
                  <normaloff>:/images/Icon_193.ico</normaloff>:/images/Icon_193.ico</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="DeleteAdvance">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="rc.qrc">
                  <normaloff>:/images/Icon_194.ico</normaloff>:/images/Icon_194.ico</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>SACCOs</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>SACCOs where this employee is a member:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeView" name="SaccoTree"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="NewSacco">
                <property name="text">
                 <string>Join New SACCO</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="SaveSacco">
                <property name="text">
                 <string>Save Changes</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="RemoveSacco">
                <property name="text">
                 <string>Remove Sacco</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Details</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="1" column="0">
                <widget class="QLabel" name="label_26">
                 <property name="text">
                  <string>Date Joined:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDateEdit" name="SaccoDateJoined">
                 <property name="displayFormat">
                  <string>dd-MMM-yyyy</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_28">
                 <property name="text">
                  <string>Sacco:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="3">
                <widget class="QComboBox" name="SaccoList"/>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_27">
                 <property name="text">
                  <string>Monthly Contribution:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QDoubleSpinBox" name="SaccoMonthlyContribution">
                 <property name="maximum">
                  <double>99990000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_New"/>
   <addaction name="actionSave"/>
   <addaction name="actionDeactivate"/>
   <addaction name="actionIssue_Advance"/>
  </widget>
  <action name="actionAdd_New">
   <property name="text">
    <string>1   Add New</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>2   Save</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionDeactivate">
   <property name="text">
    <string>3   Deactivate</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionIssue_Advance">
   <property name="text">
    <string>4   Issue Advance</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>EmployeeNo</tabstop>
  <tabstop>FirstName</tabstop>
  <tabstop>OtherNames</tabstop>
  <tabstop>IDNo</tabstop>
  <tabstop>PINNo</tabstop>
  <tabstop>NHIFNo</tabstop>
  <tabstop>NSSFNo</tabstop>
  <tabstop>DateOfBirth</tabstop>
  <tabstop>DateEmployed</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>Department</tabstop>
  <tabstop>JobTitle</tabstop>
  <tabstop>Supervisor</tabstop>
  <tabstop>Address</tabstop>
  <tabstop>Tel</tabstop>
  <tabstop>Name</tabstop>
  <tabstop>Relationship</tabstop>
  <tabstop>Tel_2</tabstop>
  <tabstop>BasicPay</tabstop>
  <tabstop>HouseAllowance</tabstop>
  <tabstop>PaymentMethod</tabstop>
  <tabstop>BankName</tabstop>
  <tabstop>BankBranch</tabstop>
  <tabstop>AccountName</tabstop>
  <tabstop>AccountNo</tabstop>
  <tabstop>AdvancesTree</tabstop>
  <tabstop>AddAdvance</tabstop>
  <tabstop>DeleteAdvance</tabstop>
  <tabstop>SaccoTree</tabstop>
  <tabstop>NewSacco</tabstop>
  <tabstop>SaveSacco</tabstop>
  <tabstop>RemoveSacco</tabstop>
  <tabstop>SaccoList</tabstop>
  <tabstop>SaccoDateJoined</tabstop>
  <tabstop>SaccoMonthlyContribution</tabstop>
 </tabstops>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
